virt/virt-2-4-release-notes.adoc

```diff
@@ -132,20 +127,11 @@ than a unique one from a pool. Although rare, randomly assigned MAC addresses ca
 from the CLI by running the following `oc` patch command:
 +
 ----
-$ export TARGET_NAMESPACE=openshift-cnv CNV_CHANNEL=2.3 && oc patch -n "${TARGET_NAMESPACE}" $(oc get subscription -n ${TARGET_NAMESPACE} --no-headers -o name) --type='json' -p='[{"op": "replace", "path": "/spec/channel", "value":"'${CNV_CHANNEL}'"}, {"op": "replace", "path": "/spec/installPlanApproval", "value":"Automatic"}]'
```

Comment:
This should still be reviewed by QE but the version still needs to be updated to 2.4 for this release